### ISSUE TITLE
Bug in yaml file : description with "à"

### ISSUE THREAD

**Issue by ChoiZ** [yomguy/DeeFuzzer#20]
In my yaml file my stream description contain "à" character and it's fail.

```bash
$ deefuzzer deefuzzer.yaml
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 552, in __bootstrap_inner
    self.run()
  File "/usr/local/lib/python2.7/dist-packages/DeeFuzzer-0.6.6-py2.7.egg/deefuzzer/core.py", line 122, in run
    self.stations.append(Station(station, q, self.logger, self.m3u))
  File "/usr/local/lib/python2.7/dist-packages/DeeFuzzer-0.6.6-py2.7.egg/deefuzzer/station.py", line 122, in __init__
    self.channel.description = self.station['infos']['description']
TypeError: String argument required
```
```yaml
    infos: {name: "station name",
            description: "Test là",
            genre: music,
            url: 'http://www.station.fm'}
```
If I replace "là" by "la", it's works well.

Is yaml string are in UTF-8? I haven't test with xml, I can test it ;-).

**Comment by ChoiZ** [comment]
I'll check this too.

**Comment by ChoiZ** [comment]
I confirm. No crash but nasty chars on Icecast metadata.

**Comment by yomguy** [comment]
In ISO or UTF8 mode, the special chars are displayed bably. The only way I managed to tune this is to force icecast to run the mountpoint in UTF8 like that:
```
  <mount>
   <mount-name>/test.mp3</mount-name>
   <charset>utf8</charset>
  </mount>
```

**Comment by achbed** [comment]
I have my Icecast server set up that way and it seems to be OK.  I had done that during my initial configuration ages ago and forgot all about it.  It's a good note - maybe we should make a note of this in the wiki?

**Comment by yomguy** [comment]
Me too :) Sure, I'll had this to the doc.
@achbed, are you OK with 0.7?

**Comment by achbed** [comment]
See comment on (closed) #66.  That's my only open question at the moment for 0.7.  We do have #67 we should address before we declare everything good.
